common/cuda_hip/factorization/cholesky_kernels.hpp.inc

```diff
@@ -156,3 +156,36 @@ __launch_bounds__(default_block_size) void cholesky_symbolic_factorize_kernel(
         out_cols[out_base] = row;
     }
 }
+
+
+template <typename ValueType, typename IndexType>
```

Comment:
I think this is from the cholesky PR ?

Reply:
no, this code was only moved from cuda/factorization/cholesky_kernels.hpp so we have a single place where we set the stream for CUDA and HIP kernels. The functions were identical except for the kernel call syntax anyways.

Reply:
but thanks for pointing this out, it's maybe not entirely clear with the files being so apart in the listing 😅